src/services/regenerate-document-task-processor.spec.js

```diff
@@ -0,0 +1,56 @@
+import sinon from 'sinon';
```

Comment:
I'd go for `document-regeneration-task-processor.js`, equivalent to `document-import-task-processor.js`

Reply:
Done